backend/.env.example

```
@@ -1,24 +1,32 @@
 POSTGRES_PASSWORD=alienpassword
 POSTGRES_DB=alien_gateway
 POSTGRES_PORT=5432
+# Full connection string used by the TypeORM data source
 DATABASE_URL=postgres://alien:alienpassword@localhost:5432/alien_gateway
 
-# API Authentication
-API_KEYS=key1,key2,key3 # Comma-separated list of valid API keys
+# --- Stellar / Soroban Configuration ---
+# RPC endpoints for interacting with the Soroban network
+STELLAR_RPC_URL=https://soroban-testnet.stellar.org
+SOROBAN_RPC_URL=https://soroban-testnet.stellar.org 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)